cloudformation/application-setup-basic/stacks/application/application.yml

```diff
@@ -0,0 +1,50 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
This is a new parent stack that combines the network and service stacks under a single parent stack. This way we don't have to manage the network and service stacks separately. I would have liked to include the cognito stack, but it doesn't expose the client secret, so I had to take it out and create it before this one.